src/common/environment.ts

```diff
@@ -2,4 +2,8 @@ export class Env {
     public static get() {
         return process.env.NODE_ENV;
     }
+
+    public static isDevelopment(): boolean {
```

Comment:
This may be a little misleading. There are more envs than just dev and prod (I know there's test, but there may be more?). Maybe instead something like `isProd` and then `!` it when running the check

Reply:
I would also only add these middlewares in dev.  So that in tests and prod they aren't registered.  In test run they log a lot of unneeded noise.